docs/en/observability/metrics-threshold-alert.asciidoc

```diff
@@ -1,5 +1,8 @@
 [[metrics-threshold-alert]]
 = Create a metrics threshold rule
+++++
+<titleabbrev>Metrics threshold</titleabbrev>
```

Comment:
Should these be Metric instead of Metrics? Looks like that's how it is in the UI. 
![image](https://github.com/elastic/observability-docs/assets/114418652/207761fd-e68f-411e-ad34-44b71a98f504)

Reply:
Good catch!